torch/onnx/symbolic_opset9.py

```diff
@@ -1412,17 +1411,20 @@ def symbolic_fn(
                 mode_s="constant",
                 value_f=0.0,
             )
-            padding = (0,) * len(padding)
+            result_padding = (0,) * len(padding)
         if ceil_mode:
-            padding = padding + tuple(a + b for (a, b) in zip(padding_ceil, padding))
+            padding_ceil = get_pool_ceil_padding(input, kernel_size, stride, padding)
+            result_padding = padding + tuple(
```

Comment:
just by looking at the code, there is potential behavior change. If in old code `padding` was modified in above `count_include_pad` branch, it is now captured in `result_padding`.

Reply:
Good catch will update

Reply:
Updated